src/routes/about/index.svelte

```diff
@@ -549,6 +549,31 @@
           committee members who provided valuable feedback throughout several
           iterations of updates.
         </p>
+        <p>
+          Cal-Adapt is evolving substantially, expanding beyond its current
+          capabilities to support <a
+            href="https://opr.ca.gov/climate/icarp/climate-assessment/"
+            target="_blank">California’s Fifth Climate Change Assessment</a
+          >
+          and offer a more comprehensive and powerful solution for highly technical
+          & data-intensive needs, with a focus on the energy sector. The expanded
```

Comment:
Again maybe replace the '&' with 'and'

Reply:
Fixed in https://github.com/berkeley-gif/caladapt-website-2021/pull/256/commits/32c481bd1634ca33062de3ee9a0ab98528807f34